b2share/modules/records/minters.py

```diff
@@ -93,7 +93,7 @@ def b2share_pid_minter(rec_pid, data):
     throw_on_failure = current_app.config.get('CFG_FAIL_ON_MISSING_PID', True)
 
     try:
-        pid = createHandle(url)
+        pid = create_handle(url)
```

Comment:
At some point we should register the HANDLE PIDs after the commit in an asynchronous task. Otherwise we will have from time to time PIDs which are registered but the transaction rollbacks because of some error.

Reply:
Agree. Make an issue?